app/services/discourse_translator/category_translator.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module DiscourseTranslator
+  class CategoryTranslator
+    # unlike post and topics, categories do not have a detected locale
+    # and will translate two fields, name and description
+
+    def self.translate(category, target_locale = I18n.locale)
+      return if category.blank? || target_locale.blank?
+
+      # locale can come in various forms
+      # standardize it to a _ symbol
+      target_locale_sym = target_locale.to_s.sub("-", "_").to_sym
+
+      translator = DiscourseTranslator::Provider::TranslatorProvider.get
+      translated_name = translator.translate_text!(category.name, target_locale_sym)
+      translated_description = translator.translate_text!(category.description, target_locale_sym)
+
+      category.update!(name: translated_name, description: translated_description)
```

Comment:
My understanding is that in a job we loop through locale, translate and then update name and description column. So if we have SiteSetting.automatic_translation_target_languages = "en|pl|de", only last locale translation will be saved. 

Is it expected?